lightspeed/modules/proc_configure-vscode-extension.adoc

```diff
@@ -19,7 +19,7 @@ You can configure the Ansible VS Code extension to enable Ansible Lightspeed, an
 . Select *Ansible Lightspeed* settings, and specify the following information:
 .. Select the *Enable Ansible Lightspeed* check box.
 .. In the *URL for Ansible Lightspeed* field, verify that you have the following URL: `\https://c.ai.ansible.redhat.com/`.
-. Optional: If you want to use a different {ibmwatsonxcodeassistant} model, enter the model ID of the model you want to use in the *Model ID Override* field. 
+. Optional:  If you want to use the custom model instead of the default model, in the *Model ID Override* field, enter the custom model ID. The *model-override* setting enables you to override the default model and use the custom model, after your organization administrator has created a custom model and has shared the model ID with you separately. 
```

Comment:
I think there's a double space after Optional: at the start of the sentence

Reply:
You're right, and thanks for catching that. Updated the spacing.